### DISCUSSION TITLE
What replaced Switch's unmount of components??

### DISCUSSION THREAD

**Discussion by aarondig** [remix-run/react-router#8625] [Q&A]
In the single paragraph of documentation regarding the most important thing in this update, you mention Route has replaced Switch. I can't find anywhere how to unmount a component on route change.

**Comment by mfpopa** [comment]
`Switch` has been replaced by `Routes` not `Route`.

```jsx
<Routes>
    <Route path='/routeX' element={<PageX />} />
    ...
<Routes>
```

You do not need to unmount anything. When the route changes, the React components mounted by the current route will be automatically unmounted.

**Reply by JustynaKK** [reply]
Hello, and is there a way to stop the route from unmounting? In my app I would need to keep react state between switching the routes

**Reply by bhushan-patil-official** [reply]
Any luck for stopping the unmounting of component on route change?
